src/umessage/umessagebuilder.rs

```diff
@@ -70,7 +37,7 @@ pub struct UMessageBuilder {
     permission_level: Option<u32>,
     comm_status: Option<EnumOrUnknown<UCode>>,
     request_id: Option<UUID>,
-    payload: Option<Bytes>,
+    payload: Option<Vec<u8>>,
```

Comment:
what is the reason for changing this? FMPOV Bytes is much more flexible when it comes to writing/reading and does require much less copying ...